### PR TITLE
datafactory t2 readme config

### PR DESCRIPTION
**Pull request by RAY-316**
https://github.com/Azure/sdk-release-request/issues/1441

### CODE REVIEW

**Comment by msyyc** [comment]
the breaking change for python is ok

**Comment by msyyc** [comment]
@qiaozha @allenjzhang @ArcturusZhang To generate track2 python SDK, we have to change swagger. Could you help check whether the swagger change influence your SDK?

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/ManagedPrivateEndpoint.json [hunk 1]
```diff
@@ -54,7 +54,10 @@
       },
       "additionalProperties": {
         "type": "object"
-      }
+      },
+      "required": [
+        "type"
+      ]
```
Comment (ArcturusZhang) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/ManagedPrivateEndpoint.json:60]: We are setting `type` as required here, but I did not find a property in this model with name of `type`. How could this happen?

Comment (ArcturusZhang) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/ManagedPrivateEndpoint.json:60]: In addition, this model has marked `type` as discriminator, which requires us to make the property `type` required. But still, I did not find this property in this model. Therefore we might need more fix to make this swagger reasonable

Comment (msyyc) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/ManagedPrivateEndpoint.json:60]: the change is the advice of @lmazuel . And I also sent e-mail called 'RE: AutoRest error about azure-sdk-for-python-track2'. Do you think we need make a quick meeting to discuss about it?

**Review by msyyc — APPROVED** [review]

**Review by msyyc — APPROVED** [review]

**Comment by msyyc** [comment]
@akning-ms We only edit 'readme.python.md' and it should not influence other SDK. Please help merge the PR

**Pull request merged**